e2e/testdata/fn-eval/out-of-place-dir/.expected/exec.sh

```diff
@@ -0,0 +1,28 @@
+#! /bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+# create a temporary directory
+TEMP_DIR=$(mktemp -d)
+
+kpt fn eval --image gcr.io/kpt-fn/set-namespace:v0.1.3 -o $TEMP_DIR -- namespace=staging
+
+# copy back the resources
+rm -r ./*
+cp $TEMP_DIR/* .
```

Comment:
This tells me we badly need `out-of-place` support in the test harness (will see if I can plug it in or pull @Shell32-Natsu to help here)

Reply:
Yes, it's a hack that needs to be addressed in separate PR

Reply:
I played a little with the test harness and was able to test it out. Here is the draft PR:
https://github.com/GoogleContainerTools/kpt/pull/2163

Reply:
@phanimarupaka I think we have an agreement here, pl. pick the change from my PR that will eliminate the need for this script completely.

Reply:
Updated.